war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -1286,6 +1286,11 @@ function rowvgStartEachRow(recursive, f) {
     registerRegexpValidator(e, /./, "Field is required");
   });
 
+  // validate required form values
+  Behaviour.specify("INPUT.not-blank", "input-not-blank", ++p, function (e) {
```

Comment:
have you considered changing the behaviour of the required validator?

Is it likely that you would ever want to allow just whitespace with it?

Reply:
I also thought that maybe we can just change the `required` validator but wanted to go the safe way for now. Technically it doesn't stop one from submitting anyway. So maybe we can really just change the `required` validator.